core/static/css/profile.css

```diff
@@ -0,0 +1,17 @@
+* {
+  margin:0;
+  padding:0;
+  text-rendering: optimizeLegibility;
+  font-family: sans-serif;
+}
+
+.headings {
+  float:left;
+  width: 100px;
+  font-weight: bold;
+}
+
+.details {
+  float:left;
+  width:300px;
+}
```

Comment:
We should have SASS processing built into this project. Have you used SASS before? If so, I'd like to use that instead of CSS and figure out how the asset pipeline works now.